ecf/scripts/workflow_manager/cycled/gdas/jgdas_emc_arch.ecf

```diff
@@ -0,0 +1,67 @@
+#PBS -S /bin/bash
+#PBS -N %RUN%_emc_arch_%CYC%
+#PBS -j oe
+#PBS -q %QUEUE_ARCH%
+#PBS -A %PROJ%-%PROJENVIR%
+#PBS -l walltime=06:00:00
+#PBS -l place=vscatter,select=1:ncpus=1:mem=1GB
+#PBS -l debug=true
+
+model=gfs
+%include <head.h>
+%include <envir-p1.h>
+
+set -x
+
+export NET=%NET:gfs%
+export RUN=%RUN%
+export CDUMP=%RUN%
+
+############################################################
+# Load modules
+############################################################
+
+module list
+
+#############################################################
+# WCOSS environment settings
+#############################################################
+export cyc=%CYC%
+export cycle=t%CYC%z
+export EXPDIR=${EXPDIR:-$HOMEgfs/parm/config}
+%include <emc_cdate.h>
```

Comment:
The vrfy and arch jobs has been removed from this PR because Arun requested it will be in a different PR. And the scope of the vrfy and arch JJobs is now been changed to include modification of rocoto arch.sh and vrfy.sh files pending rocoto workflow test.